src/Menubar.ts

```diff
@@ -290,6 +290,8 @@ export class Menubar extends EventEmitter {
 			...defaults,
 			...this._options.browserWindow,
 		});
+		
+		this.emit('after-create-window');
```

Comment:
Instead of `real-...`, we could maybe call this  `after-new-browser-window`? to be explicit.

Could you also update the README event emitter?